db/migrate/20221123133750_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.integer :authorId
+      t.integer :postId
```

Comment:
- [ ] Kindly avoid using camel cases for your attribute names. Based on convention this should be the following. Kindly fix that.

  ```
  t.integer :author_id
  t.integer :post_id
  ```

- [ ] Ensure these changes are reflected in the `schema.rb` file.